.env.local.production

```
@@ -0,0 +1,2 @@
+# Created by Vercel CLI
+VERCEL_OIDC_TOKEN="eyJhbGciOiJSUzI1NiIsInR5cCI6IkpXVCIsImtpZCI6Im1yay00MzAyZWMxYjY3MGY0OGE5OGFkNjFkYWRlNGEyM2JlNyJ9.eyJpc3MiOiJodHRwczovL29pZGMudmVyY2VsLmNvbS9odXN0bGUtbGF1bmNoIiwic3ViIjoib3duZXI6aHVzdGxlLWxhdW5jaDpwcm9qZWN0OmNpdGF0aW9uLW1hbmFnZXI6ZW52aXJvbm1lbnQ6ZGV2ZWxvcG1lbnQiLCJzY29wZSI6Im93bmVyOmh1c3RsZS1sYXVuY2g6cHJvamVjdDpjaXRhdGlvbi1tYW5hZ2VyOmVudmlyb25tZW50OmRldmVsb3BtZW50IiwiYXVkIjoiaHR0cHM6Ly92ZXJjZWwuY29tL2h1c3RsZS1sYXVuY2giLCJvd25lciI6Imh1c3RsZS1sYXVuY2giLCJvd25lcl9pZCI6InRlYW1fd3dSOWRhVkRiaGNBSXhyejRHdERYS3NJIiwicHJvamVjdCI6ImNpdGF0aW9uLW1hbmFnZXIiLCJwcm9qZWN0X2lkIjoicHJqX2I5cE5ZbmpoZnVwZ3o1RzdNdGRPbXNCd1VuYm8iLCJlbnZpcm9ubWVudCI6ImRldmVsb3BtZW50IiwicGxhbiI6InBybyIsInVzZXJfaWQiOiI5Zmw1bnBvNWo0M3RzakhoVDVyNGE5TWIiLCJuYmYiOjE3NzUwMzM1NDQsImlhdCI6MTc3NTAzMzU0NCwiZXhwIjoxNzc1MDc2NzQ0fQ.OLwWAaqAWj_QvShQwA-9RciRyyynPhuCzWNdo-9Hejice0vSvkirsgoYxFlmRcx1eh2imwfMXKacZ7lz2jYF8a_vYarbfocNMfmMyyAgUxGFxwqHwOueQYsHE3_gMSePaERHhbI7e932pNZofrpjAQ2o9gq2cTnHFRqjnK3lxg816sxlfX4xBPBjQKM6yx3pmPql4AQ_o9ntgQKy0VgP746MDA6tEJ7ZCse09tDnWcVZ_nmQwzekr-R8p3Ugub7ITNIwNJMPwvVHVeMAci4gcRttyRgaay9QGmGn22WnZvTT5E24iAO-J8A4Cz69r33QypnK4dD-GOJKeO77ZcM6Kw"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)